demo/src/screens/componentScreens/TextFieldScreen/InputsScreen.js

```diff
@@ -41,22 +41,22 @@ export default class InputsScreen extends Component {
       message = 'Please enter a valid text';
     }
     this.setState({error: message});
```

Comment:
If there is a dedicated screen for validations why not clean this screen from all error examples to keep it simple?

Reply:
yes, that's the plan.. that what I will do afterwards.. 
I just wanted to start simple and create the different screens without changing the current one.. I plan to clean the general screen (it has too many examples)

Reply:
Cool. So you want to merge it like that or after the cleanup?

Reply:
Done. I'll make it more simple in the future, for now I move the error/validation examples